src/node/in_memory.rs

```diff
@@ -1217,23 +1211,7 @@ impl<S: ForkSource + std::fmt::Debug + Clone> InMemoryNode<S> {
                 )
             );
 
-            let publish_block_l1_bytes = BLOCK_OVERHEAD_PUBDATA;
-            tracing::info!(
-            "Publishing full block costs the operator up to: {}, where {} is due to {} bytes published to L1",
```

Comment:
are you sure we need to remove it? it may still be valuable info, though I am a bit out of context

Reply:
I removed it, because it's no longer calculated by the bootloader. But I will take a look, maybe I will add some extra data about the pubdata/execution overhead